modules/docs/src/Volo.Docs.Admin.Web/DocsAdminWebModule.cs

```diff
@@ -46,6 +53,26 @@ public override void ConfigureServices(ServiceConfigurationContext context)
             {
                 options.AddProfile<DocsAdminWebAutoMapperProfile>(validate: true);
             });
+
+            Configure<AbpBundlingOptions>(options =>
+            {
+                options
+                    .ScriptBundles
+                    .Get(StandardBundles.Scripts.Global)
+                    .AddFiles("/libs/signalr/signalr.js")
+                    .AddFiles("/Pages/Docs/Admin/docs-notification-hub.js");
```

Comment:
I think adding signalr to the global scripts is not a good idea. Add it to only the pages we are using it.
Also, `/Pages/Docs/Admin/docs-notification-hub.js` should only be inside a specific page, not in global scripts.